app/controllers/spotlight/exhibits_controller.rb

```diff
@@ -89,8 +89,7 @@ def exhibit_params
         :description,
         :published,
         :tag_list,
-        contact_emails_attributes: [:id, :email],
-        languages_attributes: [:id, :public]
```

Comment:
@mejackreed I think removing this line is affecting the Current Languages form. I can't update the language to "Public" now.

Reply:
Interesting! It passes this test because it gets the  `
The exhibit was successfully updated` flash message. But when the page reloads the box is still unchecked and the ActiveRecord object is unchanged.

https://github.com/projectblacklight/spotlight/blob/21f62f6ce7edbd70cf6bd5440b2d69484a2ee6ad/spec/features/exhibits/language_create_edit_spec.rb#L28-L32

Reply:
Nice catch! Maybe we can do like a `expect(exhibit.languages.last).to be_public` or something like that as well.

Reply:
Awesome catch @camillevilla , thanks for the recommendation @jkeck . Test added in dbc700e7ba1f76de21f1d7fa90448606793ec2a7 and an update removing this change (accidental) in 2059a7bd626815c13f5af3e8faf576b43d4777a2